content/news/028/index.md

```diff
@@ -74,6 +74,58 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library Updates
 
+### [godot-rust](https://github.com/godot-rust/godot-rust)
+
+![module before/after](godot-rust-modules.png)
+
+godot-rust ([GitHub][gd-github], [Discord][gd-discord], [Twitter][gd-twitter])
+is a Rust library that provides bindings for the Godot game engine.
+
+November has been a month of refactoring for godot-rust. The API was cleaned up
+across different locations, reducing confusion and making the library more
+accessible. The module simplification ([#811][gd-811]) continued initial efforts
+on the
+module structure, such as shorter paths and avoidance of redundant re-exports.
+The improvements are best expressed as a picture -- these are differences
+between version v0.9.3 and now:
```

Comment:
This line doesn't make sense now that the picture is at the top - I think it'd read fine if we removed it?